storage/range_test.go

```diff
@@ -2366,10 +2366,11 @@ func TestRangeDanglingMetaIntent(t *testing.T) {
 	rlArgs.IgnoreIntents = true
 	var origCount, newCount int
 	for i := 0; i < 100; i++ {
-		rlArgs.Timestamp = proto.ZeroTimestamp
+		clonedLRArgs := gogoproto.Clone(rlArgs).(proto.InternalRangeLookupRequest)
+		clonedLRArgs.Timestamp = proto.ZeroTimestamp
 		rlReply.Reset()
```

Comment:
Should we make a new reply too? it doesn't seem to actually cause a race in this case, but it just looks odd to reuse a reply buffer while carefully cloning the request.